cpp/arcticdb/version/version_map.hpp

```diff
@@ -246,15 +246,6 @@ class VersionMapImpl {
             log_write(store, key.id(), key.version_id());
     }
 
-    AtomKey write_tombstone_all_key(
```

Comment:
This was unused.

Reply:
In the old internal project it was used in replication, not sure if you ripped it out there too though?

Reply:
Good memory! Enterprise uses,

```
bool delete_version_and_previous_versions(
        const std::shared_ptr <Store> &store,
        const std::shared_ptr <VersionMap> &version_map,
        const StreamId &stream_id,
        VersionId version_id) {
    ARCTICDB_SAMPLE(deleteVersionAndPreviousVersions, 0)
    auto dummy_key = atom_key_builder()
        .version_id(version_id)
        .build(stream_id, KeyType::VERSION);
    version_map->tombstone_from_key_or_all(store, stream_id, dummy_key);
    auto symbol_deleted = !has_undeleted_version(store, version_map, stream_id);
    return symbol_deleted;
}
```

now, so I think should survive this change.